src/stream-chat-net/stream-chat-net.csproj

```diff
@@ -1,58 +1,40 @@
-﻿<Project ToolsVersion="16.5" Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Label="Configuration">
     <DefineConstants>TRACE;DEBUG;NETCORE</DefineConstants>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>net45;net461;net472;netcoreapp2.1;netcoreapp3.1;net5.0;netstandard1.6;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net45;net46;net47;net48;netstandard1.6;netstandard2.0;netstandard2.1;net5.0</TargetFrameworks>
```

Comment:
dropping core 2.1 and 3.1 

are there any drawbacks ?

Reply:
no, `netcoreapp` never should have been there in the first place. `.netstandard` already covers `netcoreapp`.

Reply:
great then